tests/basic/commands

```diff
@@ -19,17 +19,17 @@ test tests/basic/isisLoopbacks.ref get isisLoopbacks
 test tests/basic/neighbors.ref get neighbors neighborTypes=["ebgp","ibgp","ospf","lan"]
 test tests/basic/nodes-summary.ref get nodes summary=true
 test tests/basic/nodes.ref get nodes summary=false
-test tests/basic/nodes-diff.ref get nodes summary=false | differential=true
+test tests/basic/nodes-diff.ref get nodes summary=false, differential=true
 test tests/basic/ospfLoopbacks.ref get ospfLoopbacks
 test tests/basic/ospfSessionCheck.ref get ospfsessioncheck
 test tests/basic/pairwiseVpnConnectivity.ref get pairwiseVpnConnectivity
 test tests/basic/routes.ref get routes
 test tests/basic/routes-diff.ref get routes differential=true
 test tests/basic/selfAdjacencies.ref get selfAdjacencies
-test tests/basic/traceroute-1-2.ref get traceroute ingressNode="as1core1" | dstIp="2.128.0.101"
-test tests/basic/traceroute-2-1.ref get traceroute ingressNode="host2" | dstIp="1.0.1.1"
-test tests/basic/multipath-host1.ref get reachability type="multipath" | ingressNodeRegex="host1" | srcIps=["2.128.0.0"] | dstIps=["3.0.1.2"] | ipProtocols=["TCP"] | srcPorts=[0] | dstPorts=[0]
-test tests/basic/multipath-host2.ref get reachability type="multipath" | ingressNodeRegex="host2" | srcIps=["2.128.0.0"] | dstIps=["1.0.1.1"] | ipProtocols=["UDP"] | srcPorts=[0] | dstPorts=[0]
+test tests/basic/traceroute-1-2.ref get traceroute ingressNode="as1core1", dstIp="2.128.0.101"
+test tests/basic/traceroute-2-1.ref get traceroute ingressNode="host2", dstIp="1.0.1.1"
+test tests/basic/multipath-host1.ref get reachability type="multipath", ingressNodeRegex="host1", srcIps=["2.128.0.0"], dstIps=["3.0.1.2"], ipProtocols=["TCP"], srcPorts=[0], dstPorts=[0]
```

Comment:
(Consider adding a unit test or 10 :p.

Otherwise, looks good to me when you're happy with it.)